libethash-cl/CLMiner.cpp

```diff
@@ -266,6 +266,17 @@ CLMiner::~CLMiner()
     kick_miner();
 }
 
+typedef struct {
+	uint32_t gid;
+	uint32_t mix[8];
```

Comment:
Is it possible to use `h256` type for `mix`?

Reply:
I want to create a common .h file, and opencl won't understand the 'mix' data type.